ring_doorbell/doorbot.py

```diff
@@ -36,11 +36,11 @@ def model(self):
         """Return Ring device model name."""
         if self.kind in DOORBELL_KINDS:
             return 'Doorbell'
-        elif self.kind in DOORBELL_2_KINDS:
+        if self.kind in DOORBELL_2_KINDS:
```

Comment:
Please note that the subsequent line of each elif is a `return` so using an if here will not cause the other conditions to be tested. Could you please expand on how changing these `if/elif` logic helped you to fix the oauth2 issue? If that is unrelated, could you restore the `if/elif` logic?

Reply:
Hi, I thought the next line would still be evaluated if the first condition was false? Sorry, I'm new to python so was just following the linter errors after running tox -r. Example error:
************* Module ring_doorbell.doorbot
ring_doorbell/doorbot.py:37:8: R1705: Unnecessary "elif" after "return" (no-else-return)

Are these safe to ignore then, if so, I can revert those changes and bring back the `if/elif` or is there an override to ignore the no-else-return rule for that file?

Reply:
Usually makes sense to avoid an else after a return, but in our case, we are doing other tests, so it's OK to ignore the linter for now.

Reply:
If you do a return, it's ok to start with an if again, because it has already returned otherwise